Update battery.json

takeshiU opened the pull request:
The information of FirefoxOS and Chromeapp are updated

takeshiU committed:
Update battery.json

dontcallmedom commented:
As noted in #24 we prefer to only document APIs that don't match the default browser's API.

Can you update the "chromeapps" entry to refer to "browser" for url, rather than the dated TR URL?

Thanks!

takeshiU committed:
Update battery.json

takeshiU committed:
Update battery.json

Based on the comments of Dom,
I updated "chromeapps" entry  to refer to "browzer"
